src/shogun/neuralnets/ConvolutionalFeatureMap.cpp

```diff
@@ -283,6 +287,32 @@ void CConvolutionalFeatureMap::convolve(
 			outputs.matrix+i*outputs.num_rows + outputs_row_offset,
 			m_output_height, m_output_width, false);
 
+#if defined(HAVE_LINALG_LIB)
```

Comment:
@karlnapf This is one way to take care of `m_autoencoder_position` but is inefficient. A better way would be to edit linalg convolution to tackle `m_autoencoder_position` like it's done below in the existing code. If we do the other way then I'll have to edit `VIENNACL` implementation as well.